readme.txt

```diff
@@ -129,12 +128,25 @@ Have a question for us? Reach us at security@ our domain.
 
 == Frequently Asked Questions ==
 
+Question: Does MailPoet have this feature?
+Answer: Want to know if MailPoet has a specific feature without having to install the plugin? [Try the live demo](https://demo.mailpoet.com).
+
+Question: Does MailPoet store the data of the user's subscribers?
+Answer: We do not store or keep any data from your subscribers. However, if you're sending with us, we do process all the emails to be sent. It means MailPoet only logs email addresses to ensure that our service runs effectively. We do not track anything else.
+
+Question: Is MailPoet GDPR compliant?
+Answer: Yes, we are GDPR compliant. We do not have a Data Processing Agreement though. Our privacy notice can act as an agreement and GDPR takes this into consideration. By using our service, you acknowledge our Privacy Notice.
+
+We suggest that you also add a link to your our Privacy Notice in your own Privacy Notice page. What's important is to make sure that your subscribers are aware that MailPoet delivers your email, if you send with MailPoet that is.
```

Comment:
@amine-mp Please fix this typo before I approve. 

`We suggest that you also add a link to your our Privacy Notice in your own Privacy Notice page`

Should be:

`We suggest that you also add a link to our Privacy Notice in your own Privacy Notice page`

I know it's in the original product specs too. I'll leave a note for Kim/Quentin about it. :)